creator-node/sequelize/migrations/20200911004845-allow-track-and-audiusUsers-appends.js

```diff
@@ -2,19 +2,28 @@
 
 module.exports = {
   up: async (queryInterface, Sequelize) => {
+    // Scope of the migration is:
+    // Remove trackUUID from Files and replace it with trackBlockchainId and migrate all existing values over
+    // Remove trackUUID field from Tracks table
+    // Remove UNIQUE constraint for blockchainId, trackUUID in Tracks table
+    // Add NOT NULL constraint for blockchainId in Tracks table
+    // Remove audiusUserUUID field from AudiusUsers table
+    // Remove UNIQUE constraint for audiusUserUUID in AudiusUsers table (no unique constraint on blockchainId)
```

Comment:
wow lol, there was no uniqueness constraint on audiusUsers.blockchainId?

Reply:
haha there was not before